app/assets/javascripts/application.js

```diff
@@ -0,0 +1,17 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
+// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// compiled file.
+//
+// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
+// about supported directives.
+//
+//= require jquery
+//= require jquery_ujs
+//= require turbolinks
+//= require_tree .
+= require bootstrap-sprockets
```

Comment:
Expected an identifier and instead saw '='.<br>Expected an assignment or function call and instead saw an expression.<br>'require' is not defined.<br>'bootstrap' is not defined.<br>'sprockets' is not defined.